### ISSUE TITLE
reinstate keyboard focus outline

### ISSUE THREAD

**Issue by indirectlylit** [learningequality/kolibri#1936]
In the new button, checkbox, and radio button components we lost the keyboard focus outline. Let's re-instate the basic outline that we used previously.

At some time in the future, we can come back and redesign this if desired.

**Comment by indirectlylit** [comment]
this is done. Please keep an eye out for missing keyboard focus indicators. (cc @radinamatic)
